apps/dashboard/app/views/launchers/show.html.erb

```diff
@@ -8,7 +8,10 @@
 <%= bootstrap_form_for(@script, url: submit_project_launcher_path) do |f| %>
   <% @script.smart_attributes.each do |attrib| %>
     <%# TODO generate render_format %>
-    <%= create_widget(f, attrib, format: nil, hide_fixed: false) %>
+    <%= create_widget(f, attrib, format: nil) %>
```

Comment:
I'd have to check on this - but does using `hide_fixed: true` fix this?

Reply:
`hide_fixed: true` will remove all of the fixed attributes, but that also causes them to get removed from the form and not submitted.

Reply:
> hide_fixed: true will remove all of the fixed attributes, but that also causes them to get removed from the form and not submitted.

That's too bad...  Hmmmm... batch connect applications work this way. Without looking or remembering completely - they must merge what's given in the http form with the `form.yml`, so anything that's fixed and wasn't in the http form is present after the merge.  

Can you take a look at how batch connect apps use `fixed`? Seems like we should do the same thing.